data/src/bench/matmul.rs

```diff
@@ -0,0 +1,20 @@
+use std::{io::Result, path::PathBuf};
+
+use crate::resolve_path;
+
+const ASSET: &str = "bench_matmul_v0000";
+
+/// Resolves the path to the matrix multiplication benchmark data.
+pub fn data() -> Result<PathBuf> {
```

Comment:
```suggestion
pub fn data_dir() -> Result<PathBuf> {
```

This is not returning `data` (or a path to a file with data) but a dir path.